frontend/packages/client/src/pages/Community.js

```diff
@@ -320,6 +320,7 @@ export default function Community() {
               setTotalMembers={setTotalMembers}
               onLeaveCommunity={onUserLeaveCommunity}
               onJoinCommunity={onUserJoinCommunity}
+              classNames="mt-2-mobile"
```

Comment:
needed to add this margin to center a bit the button vertically after adding space bellow
<img width="489" alt="Screen Shot 2022-07-20 at 12 26 25" src="https://user-images.githubusercontent.com/7526740/180021688-24fbb720-eda4-496b-86de-10c47a23efcb.png">